test/language/statements/for-in/head-lhs-member.js

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: >
+    Head's AssignmentExpression may be a MemberExpression
+id: sec-for-in-and-for-of-statements-static-semantics-early-errors
+es6id: 13.7
```

Comment:
I believe `sec-for-in-and-for-of-statements` and `13.7.5` fit best on the id fields. (the same applies for the other new test file)